en/docs/apis/create-and-publish-your-first-api.md

```diff
@@ -68,9 +63,6 @@ Follow this procedure to publish the API and view the published API on the API D
 
 1. Click **Publish** in the left pane.
 2. On the **Lifecycle Management** page, click **Publish**.
-
-    ![Publish the API](../assets/img/apis/publish.png){.cInlineImage-full}
-
 3. Click **Go to Developer Portal**.
```

Comment:
Shouldn't we mention about intermediate sign-in step here

Reply:
it's not there now right? SSO seems to be working.

Reply:
SSO  working, I meant no need to provide credentials, But it says shows intermediate page, `Signing you in`. So user might get confused, why sign in again.

Reply:
Ah ok. done